drivers/src/sha256.rs

```diff
@@ -216,6 +245,53 @@ impl Sha256 {
     }
 }
 
+pub struct Sha256SoftwareDriver {
```

Comment:
I think this could be implemented in the fuzzer library instead of in the driver itself, as long as the Sha256 trait is public.

Reply:
Let's do that, at least for now. I want to keep the drivers crate to just the actual hardware drivers if we can.

Reply:
Okay, I'll be pushing soon.